contracts/Example1.sol

```diff
@@ -1,7 +1,27 @@
 pragma solidity ^0.8.4;
 
 contract Example1 {
-    uint8 public day = 10;
-    uint256 public amount = 1000;
-    address public ownerAddr = 0x4e653b730367e303D30d8FB39d7350D07a60dfD6;
+
+    /*
+    original layout:
+    arg1: day -> size: 008 bits
+    arg2: amt -> size: 256 bits
+    arg3: adr -> size: 160 bits (20 bytes)
+    */
+
+    // optimization 1a: packing --> put day and address next to each other (amount gets its own mem block)
+    // optimization 1b: address + day = 160 + 8 = 168 bits
+    // -> leaves 256 - 168 = 88 bits on the block
+    // --> can use up to a uint64 for amount
+    uint8 public day;
+    uint64 public amount;
```

Comment:
uint type of amount should not be changed since the amount can overflow uint64. The expectation was to reorder and have this order which would require two slots

```
uint8 public day;
address public ownerAddr;
uint256 public amount;
```